src/handler/updateVrm.ts

```diff
@@ -0,0 +1,58 @@
+import 'dotenv/config';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import logger from '../util/logger';
+import { addHttpHeaders } from '../util/httpHeaders';
+import {
+  updateVehicle,
+  getBySystemNumberAndCreatedTimestamp, searchByAll,
+} from '../services/database';
+import { TechrecordGet } from '../models/post';
+import { formatTechRecord } from '../util/formatTechRecord';
+import { SearchResult } from '../models/search';
+import { processPatchVrmRequest } from '../processors/processVrmRequest';
+import { validateUpdateVrmRequest } from '../validators/update';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  logger.info('Amend VRM Called');
+  const isRequestInvalid: APIGatewayProxyResult | boolean = validateUpdateVrmRequest(event);
+  if (isRequestInvalid) {
+    return isRequestInvalid;
+  }
+  logger.debug('Request is Valid');
+  const systemNumber: string = decodeURIComponent(event.pathParameters?.systemNumber as string);
+  const createdTimestamp: string = decodeURIComponent(event.pathParameters?.createdTimestamp as string);
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const newVrm: string = JSON.parse(event.body ?? '').newVrm as string;
+  const currentRecord: TechrecordGet = await getBySystemNumberAndCreatedTimestamp(
+    systemNumber,
+    createdTimestamp,
+  ) as TechrecordGet;
+  const techRecords: SearchResult[] = await searchByAll(newVrm);
+  logger.info(techRecords);
```

Comment:
Can we give some info alongside a log, just a small sentence or something for context